installation-scripts-onm/MASTER_START_SCRIPT.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 echo "Master start script"
 
+echo 'export KUBECONFIG=/etc/rancher/k3s/k3s.yaml' >> /home/ubuntu/.profile
+source /home/ubuntu/.profile
```

Comment:
This script should just export the necessary variable instead of playing with the profile and sourcing it for another user this script runs as.